implementations/rust/ockam/ockam_command/src/tcp/outlet/delete.rs

```diff
@@ -9,31 +9,38 @@ use ockam_api::nodes::BackgroundNodeClient;
 use ockam_core::api::Request;
 use ockam_core::AsyncTryClone;
 
-use crate::node::NodeOpts;
 use crate::tcp::util::alias_parser;
 use crate::terminal::tui::DeleteCommandTui;
 use crate::terminal::PluralTerm;
 use crate::util::async_cmd;
-use crate::{color, docs, fmt_ok, CommandGlobalOpts, OckamColor, Terminal, TerminalStream};
+use crate::{docs, fmt_ok, CommandGlobalOpts, Terminal, TerminalStream};
+use crate::{node::NodeOpts, terminal::color_primary};
 
 const AFTER_LONG_HELP: &str = include_str!("./static/delete/after_long_help.txt");
+const LONG_ABOUT: &str = include_str!("./static/delete/long_about.txt");
 
-/// Delete a TCP Outlet
+/// Delete a TCP Outlet.
```

Comment:
Just a heads up: clap trims the stops from docstrings (commands' short docs + commands' args help text)

Reply:
great point; i tend to add trailing period, just for readability; i will strip them +1